src/AppPackageBuilder.vue

```diff
@@ -189,6 +190,10 @@ export default class AppPackageBuilder extends Vue {
       : "New Acquisition";
   }
 
+  public get isDitcoUser(): boolean {
```

Comment:
- [ ] Duplicate code. if possible, put this in the Acq package store as it is used a couple of times.